zlib-static/README.md

```diff
@@ -0,0 +1,11 @@
+# Zlib static
+
+This plan packages [zlib](http://www.zlib.net) as a static library (`libz.a`).
+
+## Usage
+
+This package is intended as a dependency only, and will not run anything.
+
+## Plan supporter
```

Comment:
Could you add yourself under CODEOWNERS.md when you get a minute?

Reply:
Done ;)